frontend/ui/data/creoptions.lua

```diff
@@ -19,6 +22,107 @@ local function enable_if_equals(configurable, option, value)
     return configurable[option] == value
 end
 
+local function showValues(configurable, option)
+    local default = G_reader_settings:readSetting("copt_"..option.name)
+    local current = configurable[option.name]
+    local value_default, value_current
+    if not option.suffix then option.suffix = "" end
```

Comment:
Just to not add stuff to the `option` sub-table, you could use:
`local suffix = option.suffix or ""`
and use that local `suffix` below